src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const century19 = 19;
+  const century20 = 20;
+  const century30 = 30;
```

Comment:
Let's make your code cleaner
```suggestion
  const CENTURY = {
    "19": 19,
    "20": 20,
    "30": 30
  };
```